lib/daru/vector.rb

```diff
@@ -1164,6 +1164,16 @@ def method_missing(name, *args, &block)
       end
     end
 
+    def cut partitions
+      partitions = partitions.to_a
```

Comment:
Isn't this method categorical data only? So shouldn't it be in the Category module?